modules/understanding-vulnerability-scores.adoc

```diff
@@ -6,26 +6,20 @@
 = Understanding vulnerability scores
 
 [role="_abstract"]
-In the RHACS portal shows a single Common Vulnerability Scoring System (CVSS) base score for each vulnerability.
+The {product-title-short} portal shows a single Common Vulnerability Scoring System (CVSS) base score for each vulnerability.
 {product-title} shows the CVSS score based on the following criteria:
 
 * If a CVSS v3 score is available, {product-title} shows the score and lists `v3` along with it.
 For example, `6.5 (v3)`.
 +
 [NOTE]
 ====
-CVSS v3 scores are only available if you are using Scanner version 1.3.5 and newer.
+CVSS v3 scores are only available if you are using the StackRox Scanner version 1.3.5 and newer or Scanner V4.
 ====
 * If a CVSS v3 score is not available, {product-title} shows only the CVSS v2 score.
 For example, `6.5`.
 
 You can use the API to get the CVSS scores.
-If CVSS v3 information is available for a Common Vulnerabilities and Exposures (CVE), the response includes both CVSS v3 and CVSS v2 information.
+If CVSS v3 information is available for a vulnerability, the response includes both CVSS v3 and CVSS v2 information.
 
-For some CVEs, the Red Hat Security Advisory (RHSA) CVSS score may differ from the CVSS score visible in the RHACS portal.
-This difference is because one RHSA can contain multiple CVEs, and Red Hat sometimes assigns a different score based on how a vulnerability affects other Red Hat products.
-
-In such cases, {product-title}:
-
-* Finds the highest-scoring CVE from the National Vulnerability Database (NVD) and shows its score as the CVSS score for the RHSA.
-* Breaks out each CVE in the RHSA as a separate vulnerability with its original CVSS score (from the NVD), so that you can view each one and create policies for specific CVEs.
+For a Red Hat Security Advisory (RHSA), the CVSS score is set to the highest CVSS score among all the related CVEs. One RHSA can contain multiple CVEs, and Red Hat sometimes assigns a different score based on how a vulnerability affects other Red Hat products.
```

Comment:
I only recently learned this one, so I probably missed some instances. Suggest doing a global search for `Red Hat` and swapping for `Red{nbsp}Hat` (it keeps the line from breaking in the middle of the company name)
```suggestion
For a Red Hat Security Advisory (RHSA), the CVSS score is set to the highest CVSS score among all the related CVEs. One RHSA can contain multiple CVEs, and Red{nbsp}Hat sometimes assigns a different score based on how a vulnerability affects other Red{nbsp}Hat products.
```

Reply:
Started changing them, but there were too many. Created a ticket for the rest: https://issues.redhat.com/browse/ROX-23215